src/QMCWaveFunctions/LCAO/SoaCuspCorrection.h

```diff
@@ -50,7 +50,7 @@ class SoaCuspCorrection
   /** Maximal number of supported MOs
    * this is not the AO basis because cusp correction is applied on the MO directly.
    */
-  int MaxOrbSize;
+  int MaxOrbSize = 0;
```

Comment:
Is this defensive or is the initialization changed? (browsing on web)

Reply:
it is defensive.